src/graphql/queries/countries.js

```diff
@@ -0,0 +1,14 @@
+import { gql } from "@apollo/client";
+import { COUNTRY_FIELDS }  from "./fragments"; 
+
+export const COUNTRIES = gql`

```

Comment:
Please rename this to be something like `CountriesQuery` or `COUNTRIES_QUERY`, for now it's not clear what this conts holds inside. Also, please move `COUNTRY_FIELDS` var after the query
```suggestion
export const COUNTRIES = gql`
  query getCountries {
   countries {
    ...Country
   }  
  }
  ${COUNTRY_FIELDS}
`;
```